### PR TITLE
another attempt to fix integer overflow problem on high resolutions

### PR DESCRIPTION
**Pull request by rfomin**
Fixes this for me: https://github.com/fabiangreffrath/woof/commit/b3097103e6477eea13b78ac3c74de01445047960#r137402701

### CODE REVIEW

File: src/r_voxel.c [hunk 1]
```diff
@@ -803,7 +803,7 @@ static void VX_DrawColumn (vissprite_t * spr, int x, int y)
 	byte * dest = I_VideoBuffer + viewwindowy * linesize + viewwindowx;
 
 	// iterate over screen columns
-	fixed_t ux = (((int64_t)Ax - 1) | (FRACUNIT - 1)) + 1;
+	uint64_t ux = ((Ax - 1) | (FRACUNIT - 1)) + 1;
```
Comment (ceski-1) [inline src/r_voxel.c:806]: Unfortunately I still get flickering here (turn view slowly): [woofsav0.zip](https://github.com/fabiangreffrath/woof/files/13985964/woofsav0.zip)

Also, this causes glitches on the left side of the screen when a voxel reaches the edge.

File: src/r_voxel.c [hunk 2]
```diff
@@ -317,7 +317,7 @@ void VX_Init (void)
 
 //------------------------------------------------------------------------
 
-#define VX_MINZ         (   1 * FRACUNIT)
+#define VX_MINZ         (   4 * FRACUNIT)
```
Comment (rfomin) [inline src/r_voxel.c:320]: This eliminates flickering for me. But maybe clipping is less effective, or it depends on the resolution? I am not sure.

Comment (ceski-1) [reply src/r_voxel.c:320]: That change fixes flickering for me too, but I'll admit I'm not familiar with the voxel code.

Comment (rfomin) [reply src/r_voxel.c:320]: I've tested it further, it doesn't seem to affect performance and I don't see any problems. I've taken this value from the original `R_ProjectSprite`: https://github.com/fabiangreffrath/woof/blob/c29b89cae2a11805a1e743717e872c21aa284adc/src/r_things.c#L36
@fabiangreffrath Should we merge this?

Comment (fabiangreffrath) [reply src/r_voxel.c:320]: Reasonable. Yes, please merge. Thank you!

**Pull request merged**